apps/web/playwright/bookings-apps.e2e.ts

```diff
@@ -0,0 +1,119 @@
+import { WEBAPP_URL } from "@calcom/lib/constants";
+
+import { loginUser } from "./fixtures/regularBookings";
+import type { Fixtures } from "./lib/fixtures";
+import { test } from "./lib/fixtures";
+
+const goToApps = async (
+  page: Fixtures["page"],
+  users: Fixtures["users"],
+  bookingPage: Fixtures["bookingPage"]
+) => {
+  await loginUser(users);
+  await page.goto(`${WEBAPP_URL}/event-types`);
+  await bookingPage.goToEventType("30 min");
+  await bookingPage.goToTab("apps");
+};
+const trackingId = "XXXXXXXXXX";
+
+test.describe("Qr code", async () => {
+  test.beforeEach(({ users }) => users.deleteAll());
+  test("Install Qr code app and check the generated result", async ({ page, users, bookingPage }) => {
+    await goToApps(page, users, bookingPage);
+    await bookingPage.installRecommendedApp("qr_code");
+    await bookingPage.fillQrCodeUrlParameters({ fillText: "test=test", users });
+  });
+});
+
+test.describe("Giphy", async () => {
+  test.beforeEach(({ users }) => users.deleteAll());
+
+  test("Install Giphy app and add a gif from search", async ({ page, users, bookingPage }) => {
+    await goToApps(page, users, bookingPage);
+    await bookingPage.installRecommendedApp("giphy");
+    await bookingPage.addGifFromSearch("test");
+    await bookingPage.assertSelectedGif();
+  });
+
+  test("Install Giphy app and add a gif from link", async ({ page, users, bookingPage }) => {
+    await goToApps(page, users, bookingPage);
+    await bookingPage.installRecommendedApp("giphy");
+    await bookingPage.addGifFromLink("https://media.giphy.com/media/2tgkWwcDBISej0C5Vg/giphy.gif");
+    await bookingPage.assertSelectedGif();
+  });
+});
```

Comment:
App related code should stay within the app. See apps/routing-forms/playwright/tests e.g.